include/torch-mlir/Dialect/Torch/IR/TorchOps.td

```diff
@@ -1326,4 +1326,30 @@ def Torch_DtypeCalculateYieldDtypesOp : Torch_Op<"dtype.calculate.yield.dtypes",
   let hasVerifier = 1;
 }
 
+def Torch_AtenPermuteOp : Torch_Op<"aten.permute", [
+    AllowsTypeRefinement,
+    ReadOnly
+  ]> {
+  let summary = "Generated op for `aten::permute : (Tensor, int[]) -> (Tensor)`";
```

Comment:
I just c&p'd this from the generated version. Would need to adjust summary of op.